src/org/jitsi/impl/neomedia/rtp/translator/RTCPFeedbackMessageSender.java

```diff
@@ -40,11 +72,17 @@
     private final RTPTranslatorImpl rtpTranslator;
 
     /**
-     * The next (command) sequence numbers to be used for RTCP feedback message
-     * packets per SSRC of packet sender-SSRC of media source pair.
+     * The {@link RecurringProcessibleExecutor} which will periodically call
+     * call {@link KeyframeRequester#process()} and trigger their retry logic.
```

Comment:
will periodically call call

Could you please remove the extra occurrence of the word call, @gpolitis?